### ISSUE TITLE
Can this tool replace common anti-virus software?

### ISSUE THREAD

**Issue by BarryCui** [ComodoSecurity/openedr#30]
I wonder if OpenEDR can replace anti-virus software like trendmicro officescan. If so, then there must be some signature-based or other form of virus rule updates every day or week. Then how do I know if it is reliable to detect viruses and threats?

**Comment by stnert** [comment]
Yes, OpenEDR comes as a more robust option than a conventional antivirus in my view.
